onboarding.md

```diff
@@ -0,0 +1,53 @@
+# Ocean Transport Group Onboarding Checklist
+
+All new group members should go through the following steps.
+
+## Expectations and Code of Conduct
+
+- [ ] Review the group [participation guide](https://ocean-transport.github.io/guide.html) and make
+      sure you understand your expectations.
+- [ ] Read [Ryan's blog post](https://github.com/ocean-transport/guides) on collaborationn with GitHub
+      to give you an idea of how we operate.
+
+## Columbia and LDEO Admin Stuff
+
+- [ ] Confirm that you have a [working UNI](https://cuit.columbia.edu/cuit/manage-my-uni)
+- [ ] Confirm that you have a working LDEO email address
+- [ ] Introduce yourself via email to division administrator Jenny Lee (jennylee@ldeo.columbia.edu)
+      and make sure you are on the correct department mailing lists. Your email should state
+    - Your name and title
+    - That you are working with me
+    - The expected duration of your stay
+  
+## Servers, Clusters, and Accounts
+
+- [ ] Email Ryan to remind him to grant you access to Gyre and Abyssal servers (email to LDEO IT)
+- [ ] Email Ryan to remind him to grant you access to Ginsburg HPC cluster (email to CUIT)
+- [ ] Sign up for [Pangeo Cloud](https://pangeo.io/cloud.html#sign-up)
+- [ ] Sign up for GitHub. Tell Ryan your username so you can be added to the `ocean-transport` org
+- [ ] Ask Ryan invite you to the group slack
```

Comment:
Lamont slack + OCP slack?

Reply:
How does one get invited to those slacks? What's the process?